package.json

```diff
@@ -11,8 +11,10 @@
     "@testing-library/user-event": "^13.5.0",
     "axios": "^0.27.2",
     "firebase": "^9.9.1",
+    "firebaseui": "^6.0.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-firebaseui": "^6.0.0",
```

Comment:
I'm thinking we're going to be better off using someone else's fork of this repo so it handles React 18 nicely.

Reply:
Screw it, I added the manual file and we'll ignore the library until they update things. It's annoying a guy did the work for them and they still haven't finished the review.